pkg/utils/shutdown.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	sriovnetworkv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+)
+
+func Shutdown(c client.Client) {
+	networkList := &sriovnetworkv1.SriovNetworkList{}
```

Comment:
Can we add some log messages here?
This patch overall looks good to me.

Reply:
Added logging.
Note that the log messages might be hard to see, as the pod will be removed right after this is executed.